test/koa-sslify-test.js

```diff
@@ -1,73 +1,77 @@
-var Koa = require('koa');
+var expect = require('chai').expect;
+var koa = require('koa');
 var agent = require('supertest-koa-agent');
 var enforce = require('../index.js');
 
-describe('HTTPS not enforced', () => {
+describe('HTTPS not enforced', function () {
 
-  var app = new Koa();
+  var app = koa();
```

Comment:
OK, now I'm genuinely confused. Do you want me to add a similar option to the koa1 (master) version? I targeted the PR to koa2 because that's what I'm using and that's what my diffs are against.